searx/data/engine_descriptions.json

```diff
@@ -0,0 +1,2904 @@
+  "dailymotion":"Dailymotion on ranskalaislähtöinen Internetin videolevityssivusto. Sivusto perustettiin 15. maaliskuuta 2005, kuukausi Internetin suosituimman videolevityssivuston YouTuben perustamisen jälkeen.",
+  "vimeo":"Vimeo on Internetissä toimiva englanninkielinen videopalvelu. Se perustettiin marraskuussa 2004, mutta kenen tahansa käytettävissä olevana julkisena palveluna Vimeo avattiin 16. helmikuuta 2005. Sivusto tukee mm. videoiden näyttämistä muilla verkkosivuilla sekä pitkäaikaista videovarastointia. Videoiden katsominen ja käyttäjäksi rekisteröityminen on ilmaista. Rekisteröityneet käyttäjät voivat lisätä sivustolle omia videoitaan, kirjoittaa videoiden yhteyteen kommentteja sekä liittää profiiliinsa käyttäjäkuvan.",
+  "wikibooks":"Wikikirjasto (Wikibooks) on ilmainen internetissä julkaistava kokoelma vapaita kirjoja. Se perustuu wiki-tekniikkaan. Wikikirjastoa kirjoitetaan useilla kielillä. Sen sisältö on vapaaehtoisten kirjoittama, ja se tulee aina olemaan vapaata GNU Free Documentation -lisenssin mukaisesti. Aatteellinen Wikimedia-säätiö ylläpitää mutta ei omista Wikikirjastoa. Wikikirjaston tunnetuin sisarhanke on Wikipedia.",
+  "wikinews":"Wikiuutiset on Wikimedia Foundationin hallinnoima vapaan sisällön uutislähde. Wikiuutiset soveltaa wikiperiaatetta uutistuotantoon.",
+  "wikiquote":"Wikisitaatit on Wikimedia Foundationin hallinnoima Wikipedian sisarhanke. Se käyttää samaa MediaWiki-alustaa kuin muutkin Wikimedia Foundationin hankkeet. Wikisitaattien tarkoituksena on yhteistoiminnallisesti koota kattava valikoima eri henkilöiden sitaatteja, puheita ja kirjoituksia sekä sananlaskuja ja anekdootteja. Ajatuksen Wikisitaateista esitti Daniel Alston ja Brion Vibber toteutti asian käytännössä.",
+  "wikisource":"Wikiaineisto on Wikimedia Foundationin hallinnoima hanke, joka on tarkoitettu kaikenlaisille aiemmin julkaistuille teksteille, kuten romaaneille, kirjeille, puheille, historiallisille dokumenteille tai lakiteksteille. Edellytys on, että aineisto on tekijänoikeusvapaata joko siksi että sen tekijänoikeudet ovat rauenneet iän vuoksi tai koska se on julkaistu GNU Free Documentation Licensen alla.",
+  "wiktionary":"Wikisanakirja on Wikipedian sisarhanke, jonka tavoitteena on luoda wikipohjainen kaikkien kielten välinen vapaa sanakirja.",
+  "wikiversity":"Wikiopisto on Wikimedia Foundationin hallinnoima hanke, joka sai alkunsa Wikikirjastossa ja on laajentunut omaksi hankkeekseen.",
+  "wikivoyage":"Wikimatkat on internetissä oleva matkaopas, jota muokkaavat vapaaehtoiset käyttäjät. Tällä hetkellä se on saatavissa 21 eri kielellä.",
+  "wolframalpha":"Wolfram Alpha on Wolfram Researchin kehittelemä haku- ja vastauskone, joka julkaistiin 15.5.2009. Wolfram Alphan toiminta pohjautuu Wolfram Researchin kehittelemään Mathematica-laskentaohjelmaan tiedon etsimisessä ja käsittelyssä."
+ },
+ "fil":{},
```

Comment:
The Filipino dict is empty, but that's because the Wikipedia URL is actually [tl.wikipedia.org](https://tl.wikipedia.org/wiki/Unang_Pahina) and not fil.wikipedia.org.

Reply:
Fixed, see https://github.com/searxng/searxng/pull/333/commits/8f4f482e1d9fd0d689c3403956c6be76b815a027 : the script uses the wikipedia engine to get the right URL and HTTP headers (`zh_Hant_TW` works too) .

Also, `fetch_website_description` uses `match_language`.